### ISSUE TITLE
OSX system requirements end with 10.13

### ISSUE THREAD

**Issue by MadMattAu** [thundernest/thunderbird-website#105]
https://www.thunderbird.net/en-US/thunderbird/60.7.0/system-requirements/  does not list 10.14 which has been out for about a year.  Perhaps modify the version to "10.9 and higher" and forget the double digit version list we currently have,

Mac
Operating Systems

    Mac OS X 10.9
    Mac OS X 10.10
    Mac OS X 10.11
    Mac OS X 10.12
    Mac OS X 10.13

**Comment by Sancus** [comment]
Seems reasonable, notes/sysreqs are jorgk's wheelhouse. @jorgk3 what do you think?

**Comment by wsmwk** [comment]
I don't think "10.9 and higher" would work.  We cannot assume to immediately automatically support new versions, especially given that we are esr-only.a

Our doc is just behind the times.  https://www.mozilla.org/en-US/firefox/60.0esr/system-requirements/ lists 10.14

**Comment by MadMattAu** [comment]
Are we going to start listing builds for Windows 10?  I hope not, and the OSX builds are just as much an annual increment.  Will there be an incomparability with a Windows build or an OSX annual release in the future,  almost assuredly.  But making a manual step to update the release documentation really does not make sense.  Making a special effort when it breaks does. (as occurs with Windows.).  But I will defer to the status quo.  

Just keep in mind that I tried to find the system requirements for Firefox to answer the users question and could not.  There might be references from the ESR downloads but I can not find them, There might be in app links from the ESR version,  but I don't use that and I can find no link to system requirements from Nightly.  The system requirements do not however appear on the general download page at all.  Even now knowing it can be found in the ESR stuff I had to select enterprise,  then the ESR version I still can find no link on the site to the system requirements.
